capture_hdr_loop.py

```diff
@@ -1,68 +1,65 @@
-"""
-Capture HDR frames in a loop (while actively changing some HDR settings).
+"""Capture HDR frames in a loop (while actively changing some HDR settings).
+
+Three sets of settings are chosen. These settings provide a dynamic range of
+about 9-10 stops, with a decent low- and highlight frame which is sufficient
+to grab good data on most scenes, including shiny for a medium camera operating
+at around 0.7m-1.2m
+
 """
 
 import datetime
+from pathlib import Path
+import yaml
 import zivid
 
 
+def _get_settings(settings: zivid.Settings, path: Path):
```

Comment:
You could also probably call this function: `_read_settings_from_file(path: Path) -> zivid.Settings:`

Reply:
I wanted to, but wasn't sure how I would instantiate that type. I believe I figure it out, see next push.